### PR TITLE
Test parsing of PDRs FARs and QERs

### PR DESCRIPTION
**Pull request by EmanueleGallone**
PR's aim is to test the parsing of session rules by adding several test cases.

### CODE REVIEW

File: pfcpiface/parse-pdr_test.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+				srcIface:         access,
+				srcIfaceMask:     0xff,
+				fseID:            FSEID,
+				tunnelTEID:       teid,
+				tunnelTEIDMask:   0xffffffff,
+				farID:            farID,
+				qerIDList:        []uint32{qerID},
+				needDecap:        0x1, // OuterHeaderRemoval IE is present for uplink PDRs
+			},
+			description: "Valid Uplink PDR input with create operation",
```
Comment (osinstom) [inline pfcpiface/parse-pdr_test.go:55]:
```suggestion
			description: "Valid Uplink Create PDR input",
```

File: pfcpiface/parse-pdr_test.go [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+			expected: &pdr{
+				pdrID:        uint32(pdrID),
+				fseID:        FSEID,
+				farID:        farID,
+				srcIface:     core,
+				srcIfaceMask: 0xff,
+				ueAddress:    ip2int(UEAddress),
+				qerIDList:    []uint32{qerID},
+			},
+			description: "Valid downlink PDR input with update operation",
```
Comment (osinstom) [inline pfcpiface/parse-pdr_test.go:75]:
```suggestion
			description: "Valid downlink Update PDR input",
```

File: pfcpiface/parse-qer_test.go [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+			input: pfcpsimLib.NewQERBuilder().
+				WithID(999).
+				WithMethod(pfcpsimLib.IEMethod(create)).
+				WithQFI(0x09).Build(),
+			expected: &qer{
+				qerID: 999,
+				qfi:   0x09,
+				fseID: FSEID,
+			},
+			description: "Valid QER input with create operation",
```
Comment (osinstom) [inline pfcpiface/parse-qer_test.go:36]:
```suggestion
			description: "Valid Create QER input",
```

File: pfcpiface/parse-qer_test.go [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+			input: pfcpsimLib.NewQERBuilder().
+				WithID(999).
+				WithMethod(pfcpsimLib.IEMethod(update)).
+				WithQFI(0x09).Build(),
+			expected: &qer{
+				qerID: 999,
+				qfi:   0x09,
+				fseID: FSEID,
+			},
+			description: "Valid QER input with update operation",
```
Comment (osinstom) [inline pfcpiface/parse-qer_test.go:48]:
```suggestion
			description: "Valid Update QER input",
```

File: pfcpiface/parse-pdr_test.go (hunk 2 above)
Comment (osinstom) [inline pfcpiface/parse-pdr_test.go:75]: Would be nice to also have downlink Create PDR

File: test/integration/ies.go [hunk 5]
```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: Apache-2.0
```
Comment (osinstom) [inline test/integration/ies.go:1]: this file looks like a leftover ?

**Review by osinstom — COMMENTED** [review]
Please fix conflicts and remove leftover, then we can merge

File: test/integration/ies.go (hunk 5 above)
Comment (EmanueleGallone) [reply test/integration/ies.go:1]: yes. I forgot to sync with the main branch. Solved

**Review by osinstom — APPROVED** [review]

**Pull request merged**